iceoryx_utils/test/moduletests/test_cxx_expected.cpp

```diff
@@ -175,6 +184,15 @@ TEST_F(expected_test, VoidCreateError)
     ASSERT_THAT(sut.get_error(), Eq(12.2f));
 }
 
+TEST_F(expected_test, VoidCreateErrorAndMove)
+{
+    auto sut = expected<float>::create_error(42.0f);
+    auto movedValue = std::move(sut);
+    EXPECT_FALSE(sut.is_initialized());
+    EXPECT_TRUE(movedValue.is_initialized());
```

Comment:
Needs to be an assert otherwise we risk an invalid access.

Reply:
Fixed